<html>
<head>
    <meta charset="utf-8">
    <link rel="stylesheet" href="https://cdn.datatables.net/1.13.6/css/jquery.dataTables.css" />
    <script
          src="https://code.jquery.com/jquery-3.7.1.min.js"
          integrity="sha256-/JqT3SQfawRcv/BIHPThkBvs0OEvtFFmqPF/lYI/Cxo="
          crossorigin="anonymous"></script>
    <script src="https://cdn.datatables.net/1.13.6/js/jquery.dataTables.js"></script>
</head>
<body>

<h1>Qualiperf News & Achievments</h1>

<h2>Statistics</h2>
<table class="dataframe" id="statistics">
  <thead>
    <tr style="text-align: right;">
      <th>type</th>
      <th>count</th>
      <th>qualiperf acknowledged</th>
      <th>qualiperf not acknowledged</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>publications</td>
      <td>33</td>
      <td>22</td>
      <td>11</td>
    </tr>
    <tr>
      <td>preprints</td>
      <td>9</td>
      <td>7</td>
      <td>2</td>
    </tr>
    <tr>
      <td>submissions</td>
      <td>2</td>
      <td>1</td>
      <td>1</td>
    </tr>
    <tr>
      <td>theses</td>
      <td>5</td>
      <td>0</td>
      <td>5</td>
    </tr>
    <tr>
      <td>posters</td>
      <td>21</td>
      <td>19</td>
      <td>2</td>
    </tr>
    <tr>
      <td>presentations</td>
      <td>33</td>
      <td>26</td>
      <td>7</td>
    </tr>
    <tr>
      <td>others</td>
      <td>26</td>
      <td>4</td>
      <td>22</td>
    </tr>
  </tbody>
</table>

<h2>Publications (n=33)</h2>
<table class="dataframe" id="publications">
  <thead>
    <tr style="text-align: right;">
      <th>Title</th>
      <th>Authors</th>
      <th>Journal</th>
      <th>IF</th>
      <th>Date</th>
      <th>Qualiperf</th>
      <th>Authors Qualiperf</th>
      <th>Projects</th>
      <th>Pubmed</th>
      <th>DOI</th>
      <th>Journal_ID</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>OMEX Metadata Specification Version 1.2</td>
      <td>John Gennari, Matthias König, Goskel Misirli, Maxwell Neal, David Nickerson, Dagmar Waltemath</td>
      <td>J Integr Bioinform. 2021 Oct 20;18(3):20210020. doi: 10.1515/jib-2021-0020. PMID: 34668356; PMCID: PMC8560343.</td>
      <td>1.9</td>
      <td>2021-08-08</td>
      <td>No</td>
      <td>Matthias König</td>
      <td>P3 - König</td>
      <td>34668356.0</td>
      <td>10.1515/jib-2021-0020</td>
      <td>J Integr Bioinform</td>
    </tr>
    <tr>
      <td>The Simulation Experiment Description Markup Language (SED-ML): Language Specification for Level 1 Version 4</td>
      <td>Lucian P. Smith, Frank T. Bergmann, Alan Garny, Thomas Helikar, Jonathan Karr, David Nickerson, Herbert Sauro, Dagmar Waltemath, and Matthias König</td>
      <td>J Integr Bioinform. 2021 Oct 5;18(3):20210021. doi: 10.1515/jib-2021-0021. PMID: 35330701; PMCID: PMC8560344</td>
      <td>1.9</td>
      <td>2021-08-08</td>
      <td>No</td>
      <td>Matthias König</td>
      <td>P3 - König</td>
      <td>35330701.0</td>
      <td>10.1515/jib-2021-0021</td>
      <td>J Integr Bioinform</td>
    </tr>
    <tr>
      <td>Application of magnetic resonance imaging in liver biomechanics: a systematic review</td>
      <td>Seyedpour S, Nabati M,  Lambers L, Nafisi S, Tautenhahn HM, Sack I, Reichenbach J, Ricken T.</td>
      <td>Front Physiol. 2021 Sep 22;12:733393. doi: 10.3389/fphys.2021.733393. PMID: 34630152; PMCID: PMC8493836</td>
      <td>4.0</td>
      <td>2021-09-22</td>
      <td>Yes</td>
      <td>Hans-Michael Tautenhahn, Jürgen R. Reichenbach, Tim Ricken</td>
      <td>P9 - Tautenhahn</td>
      <td>34630152.0</td>
      <td>10.3389/fphys.2021.733393</td>
      <td>Front Physiol</td>
    </tr>
    <tr>
      <td>Model order reduction for deformable porous materials in thin domains via asymptotic analysis</td>
      <td>Armiti-Juber, Ricken T</td>
      <td>Arch Appl Mech (2021). https://doi.org/10.1007/s00419-021-01907-3</td>
      <td>2.8</td>
      <td>2020-11-15</td>
      <td>Yes</td>
      <td>Tim Ricken</td>
      <td>P6 - Reichenbach, P7 - Ricken, P9 - Tautenhahn, Coordination</td>
      <td>NaN</td>
      <td>10.1007/s00419-021-01907-3</td>
      <td>Arch Appl Mech</td>
    </tr>
    <tr>
      <td>Hepatectomy-Induced Alterations in Hepatic Perfusion and Function - Toward Multi-Scale Computational Modeling for a Better Prediction of Post-hepatectomy Liver Function</td>
      <td>Bruno Christ, Maximilian Collatz, Uta Dahmen, Karl-Heinz Herrmann, Sebastian Höpfl, Matthias König, Lena Lambers, Manja Marz, Daria Meyer, Nicole Radde, Jürgen R. Reichenbach, Tim Ricken and Hans-Michael Tautenhahn</td>
      <td>Front Physiol. 2021 Nov 18;12:733868. doi: 10.3389/fphys.2021.733868. PMID: 34867441; PMCID: PMC8637208</td>
      <td>4.0</td>
      <td>2021-11-18</td>
      <td>Yes</td>
      <td>Bruno Christ, Uta Dahmen, Karl-Heinz Herrmann, Sebastian Höpfl, Matthias König, Lena Lambers, Manja Marz, Daria Meyer, Nicole Erika Radde, Jürgen R. Reichenbach, Tim Ricken, Hans-Michael Tautenhahn</td>
      <td>P1 - Christ, P2 - Dahmen, P3 - König, P4 - Marz, P5 - Radde, P6 - Reichenbach, P7 - Ricken, P8 - Schwen, P9 - Tautenhahn, Coordination</td>
      <td>34867441.0</td>
      <td>10.3389/fphys.2021.733868</td>
      <td>Front Physiol</td>
    </tr>
    <tr>
      <td>Mesenchymal stromal cells mitigate liver damage after extended resection in the pig by modulating thrombospondin-1/TGF-β</td>
      <td>Nickel S*, Vlaic S*, Christ M*, Schubert K, Henschler R, Tautenhahn F, Burger C, Kühne H, Erler S, Roth A, Wild C, Brach J, Hammad S, Gittel C, Baunack M, Lange U, Broschewitz J, Stock P, Metelmann I, Bartels M, Pietsch U-C, Krämer S, Eichfeld U, von Bergen M, Dooley S*, Tautenhahn H-M*, Christ B*</td>
      <td>NPJ Regen Med. 2021 Dec 3;6(1):84. doi: 10.1038/s41536-021-00194-4. PMID: 34862411; PMCID: PMC8642541</td>
      <td>14.4</td>
      <td>2021-12-03</td>
      <td>Yes</td>
      <td>Hans-Michael Tautenhahn*, Bruno Christ*</td>
      <td>P1 - Christ, P9 - Tautenhahn</td>
      <td>34862411.0</td>
      <td>10.1038/s41536-021-00194-4</td>
      <td>NPJ Regen Med</td>
    </tr>
    <tr>
      <td>Semi-automated Data-driven FE Mesh Generation and InverseParameter Identification for a Multiscale and Multiphase Model ofFunction-Perfusion Processes in the Liver</td>
      <td>Lambers Lena, Mielke Andre, Ricken Tim</td>
      <td>PAMM·Proc. Appl. Math. Mech.2021;21:1 e202100190. doi: 10.1002/pamm.202100190</td>
      <td>1.7</td>
      <td>2021-09-23</td>
      <td>Yes</td>
      <td>Lena Lambers, Tim Ricken</td>
      <td>P7 - Ricken, Coordination</td>
      <td>NaN</td>
      <td>10.1002/pamm.202100190</td>
      <td>PAMM Proc Appl Math Mech</td>
    </tr>
    <tr>
      <td>Greasy hair against obesity</td>
      <td>Nickel S, Christ B</td>
      <td>Signal Transduction and Targeted Therapy (2021)6:429; https://doi.org/10.1038/s41392-021-00850-7</td>
      <td>39.3</td>
      <td>2021-12-17</td>
      <td>Yes</td>
      <td>Bruno Christ</td>
      <td>P1 - Christ</td>
      <td>34921132.0</td>
      <td>10.1038/s41392-021-00850-7</td>
      <td>Signal Transduction and Targeted Therapy</td>
    </tr>
    <tr>
      <td>Theoretical formulation and computational aspects of a two-scale homogenization scheme combining the TPM and FE2 method for poro-elastic fuid-saturated porous media</td>
      <td>Ricken Tim, Schröder Jörg, Bluhm Joachim, Maike Simon, Bartel Florian</td>
      <td>International Journal of Solid and Structures</td>
      <td>3.6</td>
      <td>2021-01-10</td>
      <td>Yes</td>
      <td>Tim Ricken</td>
      <td>P7 - Ricken</td>
      <td>NaN</td>
      <td>10.1016/j.ijsolstr.2021.111412</td>
      <td>International Journal of Solid and Structures</td>
    </tr>
    <tr>
      <td>Ramipril Reduces Acylcarnitines and Distinctly Increases Angiotensin-Converting Enzyme 2 Expression in Lungs of Rats</td>
      <td>Kosacka J*, Berger C, Ceglarek U, Hoffmann A, Blüher M, Klöting N</td>
      <td>Metabolites. 2022 Mar 26;12(4):293. doi: 10.3390/metabo12040293. PMID: 35448480; PMCID: PMC9028516.</td>
      <td>4.1</td>
      <td>2022-03-26</td>
      <td>No</td>
      <td>Joanna Kosacka</td>
      <td>P1 - Christ</td>
      <td>35448480.0</td>
      <td>10.3390/metabo12040293</td>
      <td>Metabolites</td>
    </tr>
    <tr>
      <td>Specifications of standards in systems and synthetic biology: status and developments in 2021</td>
      <td>Falk Schreiber, Padraig Gleeson, Martin Golebiewski, Thomas E. Gorochowski, Michael Hucka, Sarah M. Keating, Matthias König, Chris J. Myers, David P. Nickerson, Björn Sommer, Dagmar Waltemath</td>
      <td>J Integr Bioinform. 2021 Oct 22;18(3):20210026. doi: 10.1515/jib-2021-0026. PMID: 34674411; PMCID: PMC8573232. https://doi.org/10.1515/jib-2021-0026</td>
      <td>1.9</td>
      <td>2021-10-22</td>
      <td>Yes</td>
      <td>Matthias König</td>
      <td>P3 - König</td>
      <td>34674411.0</td>
      <td>10.1515/jib-2021-0026</td>
      <td>J Integr Bioinform</td>
    </tr>
    <tr>
      <td>Editorial: Computational Modeling for Liver Surgery and Interventions</td>
      <td>Bruno Christ, Uta Dahmen, Nicole Radde, Tim Ricken</td>
      <td>Front Physiol. 2022 Feb 15;13:859522. doi: 10.3389/fphys.2022.859522. PMID: 35242057; PMCID: PMC8886156</td>
      <td>4.0</td>
      <td>2022-04-13</td>
      <td>No</td>
      <td>Bruno Christ, Uta Dahmen, Nicole Erika Radde, Tim Ricken</td>
      <td>P1 - Christ, P2 - Dahmen, P5 - Radde, P7 - Ricken</td>
      <td>35242057.0</td>
      <td>10.3389/fphys.2022.859522</td>
      <td>Front Physiol</td>
    </tr>
    <tr>
      <td>QuaLiPerF - Multi-X Liver Modelling</td>
      <td>Tim Ricken, Lena Lambers, Bruno Christ, Uta Dahmen, Karl-Heinz Herrmann, Matthias\nKönig, Manja Marz, Nicole Radde, Jürgen R. Reichenbach, Lars Ole Schwen, and\nHans-Michael Tautenhahn</td>
      <td>GACM-Report</td>
      <td>0.0</td>
      <td>2020-12-10</td>
      <td>Yes</td>
      <td>Tim Ricken, Lena Lambers, Bruno Christ, Uta Dahmen, Karl-Heinz Herrmann, Matthias König\nKönig, Manja Marz, Nicole Radde, Jürgen R. Reichenbach, Lars Ole Schwen, and\nHans-Michael Tautenhahn</td>
      <td>P1 - Christ, P2 - Dahmen, P3 - König, P4 - Marz, P5 - Radde, P6 - Reichenbach, P7 - Ricken, P8 - Schwen, P9 - Tautenhahn, Coordination</td>
      <td>NaN</td>
      <td>-</td>
      <td>GACM-Report</td>
    </tr>
    <tr>
      <td>Streptozotocin-Induced Diabetes Causes Changes in Serotonin-Positive Neurons in the Small Intestine in Pig Model</td>
      <td>Bulc, M.; Palus, K.; Całka, J.; Kosacka, J.; Nowicki, M.</td>
      <td>Int J Mol Sci. 2022 Apr 20;23(9):4564. doi: 10.3390/ijms23094564. PMID: 35562954; PMCID: PMC9099899</td>
      <td>5.6</td>
      <td>2022-04-20</td>
      <td>No</td>
      <td>Joanna Kosacka</td>
      <td>P1 - Christ</td>
      <td>35562954.0</td>
      <td>10.3390/ijms23094564</td>
      <td>Int J Mol Sci</td>
    </tr>
    <tr>
      <td>BioSimulators: a central registry of simulation engines and services for recommending specific tools</td>
      <td>Shaikh B, Smith LP, Vasilescu D, Marupilla G, Wilson M, Agmon E, Agnew H, Andrews SS, Anwar A, Beber ME, Bergmann FT, Brooks D, Brusch L, Calzone L, Choi K, Cooper J, Detloff J, Drawert B, Dumontier M, Ermentrout GB, Faeder JR, Freiburger AP, Fröhlich F, Funahashi A, Garny A, Gennari JH, Gleeson P, Goelzer A, Haiman Z, Hasenauer J, Hellerstein JL, Hermjakob H, Hoops S, Ison JC, Jahn D, Jakubowski HV, Jordan R, Kalaš M, &lt;b&gt;König M&lt;/b&gt;, Liebermeister W, Sheriff RSM, Mandal S, McDougal R, Medley JK, Mendes P, Müller R, Myers CJ, Naldi A, Nguyen TVN, Nickerson DP, Olivier BG, Patoliya D, Paulevé L, Petzold LR, Priya A, Rampadarath AK, Rohwer JM, Saglam AS, Singh D, Sinha A, Snoep J, Sorby H, Spangler R, Starruß J, Thomas PJ, van Niekerk D, Weindl D, Zhang F, Zhukova A, Goldberg AP, Schaff JC, Blinov ML, Sauro HM, Moraru II, Karr JR.</td>
      <td>Nucleic Acids Res. 2022 Jul 5;50(W1):W108-W114. doi: 10.1093/nar/gkac331. PMID: 35524558; PMCID: PMC9252793</td>
      <td>14.9</td>
      <td>2022-05-07</td>
      <td>No</td>
      <td>Matthias König</td>
      <td>P3 - König</td>
      <td>35524558.0</td>
      <td>10.1093/nar/gkac331</td>
      <td>Nucleic Acids Res</td>
    </tr>
    <tr>
      <td>Human Mesenchymal Stromal Cells Resolve Lipid Load in High Fat Diet‐Induced Non‐Alcoholic  Steatohepatitis in Mice by Mitochondria Donation</td>
      <td>Sandra Nickel, Madlen Christ, Sandra Schmidt, Joanna Kosacka, Hagen Kühne, Martin Roderfeld,\nThomas Longerich, Lysann Tietze, Ina Bosse, Mei‐Ju Hsu, Peggy Stock, Elke Roeb, Bruno Christ</td>
      <td>Cells. 2022 Jun 2;11(11):1829. doi: 10.3390/cells11111829. PMID: 35681524; PMCID: PMC9180625</td>
      <td>6.0</td>
      <td>2022-06-02</td>
      <td>Yes</td>
      <td>Joanna Kosacka, Lysann Tietze, Bruno Christ</td>
      <td>P1 - Christ</td>
      <td>35681524.0</td>
      <td>10.3390/cells11111829</td>
      <td>Cells</td>
    </tr>
    <tr>
      <td>Model-based design of a synthetic oscillator based on an epigenetic methylation memory system</td>
      <td>Viviane Klingel, Dimitri Graf, Sara Weirich, Albert Jeltsch, Nicole E. Radde</td>
      <td>ACS Synth Biol. 2022 Jul 15;11(7):2445-2455. doi: 10.1021/acssynbio.2c00118. Epub 2022 Jun 24. PMID: 35749318; PMCID: PMC9295699</td>
      <td>4.7</td>
      <td>2022-06-02</td>
      <td>Yes</td>
      <td>Nicole Erika Radde</td>
      <td>P5 - Radde</td>
      <td>35749318.0</td>
      <td>10.1021/acssynbio.2c00118</td>
      <td>ACS Synth Biol</td>
    </tr>
    <tr>
      <td>An inverse transformation algorithm to infer parameter distributions from population snapshot data</td>
      <td>Vincent Wagner, Sebastian Höpfl, Viviane Klingel, Maria C. Pop, Nicole E. Radde</td>
      <td>IFAC-PapersOnLine, 9th IFAC Conference on Foundations of Systems Biology in Engineering FOSBE 2022, https://doi.org/10.1016/j.ifacol.2023.01.020</td>
      <td>0.0</td>
      <td>2022-06-02</td>
      <td>Yes</td>
      <td>Sebastian Höpfl, Nicole Erika Radde</td>
      <td>P5 - Radde</td>
      <td>NaN</td>
      <td>10.1016/j.ifacol.2023.01.020</td>
      <td>IFAC-PapersOnLine</td>
    </tr>
    <tr>
      <td>Multimodale Therapie bei Lebermetastasen kolorektaler Karzinome in kurativer Intention</td>
      <td>PD Dr. med. Hans-Michael Tautenhahn, Sandra Nickel, Eva Maria Kindler, Utz Settmacher</td>
      <td>Chirurgie (Heidelb). 2022 Jul;93(7):652-658. German. doi: 10.1007/s00104-022-01665-0. Epub 2022 Jun 8. PMID: 35771270</td>
      <td>0.9</td>
      <td>2022-06-08</td>
      <td>No</td>
      <td>Hans-Michael Tautenhahn, Eva Maria Kindler</td>
      <td>P9 - Tautenhahn</td>
      <td>35771270.0</td>
      <td>10.1007/s00104-022-01665-0</td>
      <td>Chirurgie</td>
    </tr>
    <tr>
      <td>Quasi-Entropy Closure: A Fast and Reliable Approach to Close the Moment Equations of the Chemical Master Equation</td>
      <td>Wagner V, Castellaz B, Oesting M, Radde N</td>
      <td>Bioinformatics. 2022 Sep 15;38(18):4352-4359. doi: 10.1093/bioinformatics/btac501. PMID: 35916726</td>
      <td>5.8</td>
      <td>2022-07-18</td>
      <td>Yes</td>
      <td>Nicole Erika Radde</td>
      <td>P5 - Radde</td>
      <td>35916726.0</td>
      <td>10.1093/bioinformatics/btac501</td>
      <td>Bioinformatics</td>
    </tr>
    <tr>
      <td>On robust discretization methods for poroelastic problems: Numerical examples and counter-examples</td>
      <td>Fleurianne Bertrand ,Maximilian Brodbeck, Tim Ricken</td>
      <td>Examples and Counterexamples,  https://doi.org/10.1016/j.exco.2022.100087</td>
      <td>0.0</td>
      <td>2022-09-28</td>
      <td>Yes</td>
      <td>Tim Ricken</td>
      <td>P7 - Ricken</td>
      <td>NaN</td>
      <td>10.1016/j.exco.2022.100087</td>
      <td>Examples and Counterexamples</td>
    </tr>
    <tr>
      <td>libRoadRunner 2.0: A High-Performance SBML Simulation and Analysis Library</td>
      <td>Ciaran Welsh, Jin Xu, Lucian Smith, Matthias König, Kiri Choi, Herbert M. Sauro</td>
      <td>Bioinformatics. 2022 Dec 8:btac770. doi: 10.1093/bioinformatics/btac770. Epub ahead of print. PMID: 36478036.</td>
      <td>5.8</td>
      <td>2022-12-08</td>
      <td>Yes</td>
      <td>Matthias König</td>
      <td>P3 - König</td>
      <td>36478036.0</td>
      <td>10.1093/bioinformatics/btac770</td>
      <td>Bioinformatics</td>
    </tr>
    <tr>
      <td>Periportal steatosis in mice affects distinct parameters of pericentral drug metabolism</td>
      <td>Albadry M.,  Höpfl, S., Ehteshamzad, N., König, M., Böttcher, M., Neumann, J., Lupp, A., Dirsch, O., Radde, N., Christ, B., Christ, M., Schwen, L. O., Laue, H., Klopfleisch, R., Dahmen, U.</td>
      <td>Sci Rep. 2022 Dec 17;12(1):21825. doi: 10.1038/s41598-022-26483-6. PMID: 36528753; PMCID: PMC9759570</td>
      <td>4.6</td>
      <td>2022-12-17</td>
      <td>Yes</td>
      <td>Mohamed Albadry, Sebastian Höpfl, Matthias König, Nicole Erika Radde, Bruno Christ, Lars Ole Schwen, Uta Dahmen</td>
      <td>P1 - Christ, P2 - Dahmen, P3 - König, P5 - Radde, P8 - Schwen</td>
      <td>36528753.0</td>
      <td>10.1038/s41598-022-26483-6</td>
      <td>Sci Rep</td>
    </tr>
    <tr>
      <td>Uncertainty with Varying Subsurface Permeabilities Reduced Using Coupled Random Field and Extended Theory of Porous Media Contaminant Transport Models</td>
      <td>S. M. Seyedpour, C. Henning, P. Kirmizakis, S. Herbrandt, K. Ickstadt, R. Doherty\nand T. Ricken</td>
      <td>Water 2023, 15, 159. https://doi.org/10.3390/w15010159</td>
      <td>3.4</td>
      <td>2022-12-31</td>
      <td>Yes</td>
      <td>Tim Ricken</td>
      <td>P7 - Ricken</td>
      <td>NaN</td>
      <td>10.3390/w15010159</td>
      <td>Water</td>
    </tr>
    <tr>
      <td>Simulation of Contaminant Transport through the Vadose Zone: A Continuum Mechanical Approach within the Framework of the Extended Theory of Porous Media (eTPM)</td>
      <td>S. M. Seyedpour, A. Thom and T. Ricken</td>
      <td>Water 2023, 15, 343. https://doi.org/10.3390/w15020343</td>
      <td>3.4</td>
      <td>2023-01-13</td>
      <td>Yes</td>
      <td>Tim Ricken</td>
      <td>P7 - Ricken</td>
      <td>NaN</td>
      <td>10.3390/w15020343</td>
      <td>Water</td>
    </tr>
    <tr>
      <td>Bayesian estimation reveals that reproducible models in Systems Biology get more citations</td>
      <td>Höpfl Sebastian, Pleiss Jürgen, Radde Nicole</td>
      <td>Sci Rep 13, 2695 (2023). https://doi.org/10.1038/s41598-023-29340-2</td>
      <td>4.6</td>
      <td>2023-02-15</td>
      <td>Yes</td>
      <td>Sebastian Höpfl, Nicole Erika Radde</td>
      <td>P5 - Radde</td>
      <td>36792648.0</td>
      <td>10.1038/s41598-023-29340-2</td>
      <td>Sci Rep</td>
    </tr>
    <tr>
      <td>Critical Evaluation of Discarded Donor Livers  in the Eurotransplant Region: Potential  Implications for Machine Perfusion</td>
      <td>Falk Rauchfuß, Hans-Michael Tautenhahn, Felix Dondorf, Aladdin Ali-Deeb, Utz Settmacher on behalf of the ELIAC</td>
      <td>e-ISSN 2329-0358 © Ann Transplant, 2023; 28: e938132 DOI: 10.12659/AOT.938132</td>
      <td>1.1</td>
      <td>2023-03-05</td>
      <td>No</td>
      <td>Hans-Michael Tautenhahn</td>
      <td>P9 - Tautenhahn</td>
      <td>36927714.0</td>
      <td>10.12659/AOT.938132</td>
      <td>Ann Transplant</td>
    </tr>
    <tr>
      <td>Specifications of Standards in Systems and Synthetic Biology: Status and Developments in 2022 and the COMBINE meeting 2022</td>
      <td>M. König, P. Gleeson, M. Golebiewski, T. Gorochowski, M. Hucka, S. Keating, C. Myers, D. Nickerson, F. Schreiber</td>
      <td>J Integr Bioinform. 2023 Mar 29;20(1):20230004. doi: 10.1515/jib-2023-0004. PMID: 36989443; PMCID: PMC10063176</td>
      <td>1.9</td>
      <td>2023-03-29</td>
      <td>Yes</td>
      <td>Matthias König</td>
      <td>P3 - König</td>
      <td>36989443.0</td>
      <td>10.1515/jib-2023-0004</td>
      <td>J Integr Bioinform</td>
    </tr>
    <tr>
      <td>Editorial: Multiscale modeling for the liver</td>
      <td>Ho H, Rezania V, Schwen LO</td>
      <td>Front. Bioeng. Biotechnol. 2023 Apr 13;1179980. doi:10.3389/fbioe.2023.1179980</td>
      <td>5.7</td>
      <td>2023-04-13</td>
      <td>No</td>
      <td>Lars Ole Schwen</td>
      <td>P8 - Schwen</td>
      <td>37122862.0</td>
      <td>10.3389/fbioe.2023.1179980</td>
      <td>Front Bioeng Biotechnol</td>
    </tr>
    <tr>
      <td>Automated Detection of Portal Fields and Central Veins in Whole-Slide Images of Liver Tissue</td>
      <td>Budelmann D, Laue H, Weis N, Dahmen U, D’Alessandro LA, Biermayer I, Klingmüller U, Ghallab A, Hassan R, Begher-Tibbe B, Hengstler JG, Schwen LO</td>
      <td>Journal of Pathology Informatics 2022 13:100001, doi: 10.1016/j.jpi.2022.100001</td>
      <td>4.3</td>
      <td>2022-02-14</td>
      <td>No</td>
      <td>(None received QuaLiPerF funding for this work)</td>
      <td>Stea-PK Mod</td>
      <td>35242441.0</td>
      <td>10.1016/j.jpi.2022.100001</td>
      <td>Journal of Pathology Informatics</td>
    </tr>
    <tr>
      <td>Perfusion and T2 Relaxation Time as Predictors of Severity and Outcome in Sepsis-Associated Acute Kidney Injury: A Preclinical MRI Study</td>
      <td>Zhao, W.-T, Herrmann, K.-H., Sibgatulin R., Nahardani A., Krämer M., Heitplatz B., Van Marck V. , Reuter S., Reichenbach J.R., Hoerr V.</td>
      <td>J Magn Reson Imaging 2023 Apr 7. doi: 10.1002/jmri.28698</td>
      <td>4.4</td>
      <td>2023-07-04</td>
      <td>No</td>
      <td>Wan-Ting Zhao, Karl-Heinz Herrmann, Jürgen R. Reichenbach</td>
      <td>P6 - Reichenbach</td>
      <td>37026419.0</td>
      <td>10.1002/jmri.28698</td>
      <td>J Magn Reson Imaging</td>
    </tr>
    <tr>
      <td>A pathway model of glucose-stimulated insulin secretion in the pancreatic β-cell</td>
      <td>Maheshvare MD., Raha S., König M.*, and Pal D.* (* equal contribution)</td>
      <td>Front. Endocrinol. 14:1185656, doi: 10.3389/fendo.2023.1185656</td>
      <td>5.2</td>
      <td>2023-02-08</td>
      <td>Yes</td>
      <td>Matthias König</td>
      <td>P3 - König</td>
      <td>37600713.0</td>
      <td>10.3389/fendo.2023.1185656</td>
      <td>Front Endocrinol</td>
    </tr>
    <tr>
      <td>Affine transformations accelerate the training of physics-informed neural networks of a one-dimensional consolidation problem</td>
      <td>Luis Mandl, André Mielke, Seyed Morteza Seyedpour, Tim Ricken</td>
      <td>Mandl, L., Mielke, A., Seyedpour, S.M. et al. Affine transformations accelerate the training of physics-informed neural networks of a one-dimensional consolidation problem. Sci Rep 13, 15566 (2023). https://doi.org/10.1038/s41598-023-42141-x</td>
      <td>4.6</td>
      <td>2023-09-20</td>
      <td>Yes</td>
      <td>Luis Mandl, Tim Ricken</td>
      <td>P7 - Ricken, SimLivA, ATLAS</td>
      <td>37730743.0</td>
      <td>10.1038/s41598-023-42141-x</td>
      <td>Sci Rep</td>
    </tr>
  </tbody>
</table>

<image src="./fig_publications.png" width="500px"></image>


<h2>Submissions (n=2)</h2>
<table class="dataframe" id="submissions">
  <thead>
    <tr style="text-align: right;">
      <th>Title</th>
      <th>Authors</th>
      <th>Journal</th>
      <th>Date</th>
      <th>Qualiperf</th>
      <th>Authors Qualiperf</th>
      <th>Projects</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>An inverse transformation algorithm to infer parameter distributions from population snapshot data</td>
      <td>*Wagner Vincent, *Höpfl Sebastian, Klingel Viviane, Pop Maria C. , Radde Nicole E.</td>
      <td>IFAC-PapersOnline series, https://www.sciencedirect.com/journal/ifac-papersonline</td>
      <td>2022-04-13</td>
      <td>Yes</td>
      <td>Sebastian Höpfl, Nicole Erika Radde</td>
      <td>P5 - Radde</td>
    </tr>
    <tr>
      <td>Segmentation of Lipid Droplets in Histological Images</td>
      <td>Budelmann D, Cao Q, Laue H, Albadry M, Dahmen U, Schwen LO</td>
      <td>NaN</td>
      <td>2023-06-04</td>
      <td>No</td>
      <td>(None received QuaLiPerF funding for this work)</td>
      <td>Stea-PK Mod</td>
    </tr>
  </tbody>
</table>

<h2>Preprints (n=9)</h2>
<table class="dataframe" id="preprints">
  <thead>
    <tr style="text-align: right;">
      <th>Title</th>
      <th>Authors</th>
      <th>Journal</th>
      <th>Date</th>
      <th>Qualiperf</th>
      <th>Authors Qualiperf</th>
      <th>Projects</th>
      <th>Pubmed</th>
      <th>DOI</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Ten Simple Rules for Sharing Experimental and Clinical Data with the Modeling Community</td>
      <td>Matthias König*, Jan Grzegorzewski, Martin Golebiewski, Henning Hermjakob, Mike Hucka, Brett Olivier, Sarah M. Keating, David Nickerson, Falk Schreiber, Rahuman Sheriff, Dagmar Waltemath</td>
      <td>Preprints 2021, 2021080303, https://www.preprints.org/manuscript/202108.0303/v1</td>
      <td>2021-08-13 00:00:00</td>
      <td>Yes</td>
      <td>Matthias König*</td>
      <td>P3 - König</td>
      <td>NaN</td>
      <td>10.20944/preprints202108.0303.v1</td>
    </tr>
    <tr>
      <td>SED-ML Validator: tool for debugging simulation experiments</td>
      <td>Bilal Shaikh, Andrew Philip Freiburger, Matthias König, Frank T. Bergmann, David P. Nickerson, Herbert M. Sauro, Michael L. Blinov, Lucian P. Smith, Ion I. Moraru, Jonathan R. Karr</td>
      <td>arXiV, 2021, 2106.00844, [Submitted to Bioinformatics 2021-06-02]</td>
      <td>2020-06-02 00:00:00</td>
      <td>Yes</td>
      <td>Matthias König</td>
      <td>P3 - König</td>
      <td>NaN</td>
      <td>10.48550/arXiv.2106.00844</td>
    </tr>
    <tr>
      <td>Dynamic Flux Balance Analysis Models in SBML</td>
      <td>Matthias König* , Leandro H. Watanabe, Jan Grzegorzewski, Chris J.\nMyers</td>
      <td>bioRxiv doi: https://doi.org/10.1101/245076</td>
      <td>2022-03-28 00:00:00</td>
      <td>Yes</td>
      <td>Matthias König, Jan Grzegorzewski</td>
      <td>P3 - König</td>
      <td>NaN</td>
      <td>10.1101/245076</td>
    </tr>
    <tr>
      <td>FAIR Sharing of Reproducible Models of Epidemic and Pandemic Forecast</td>
      <td>Ramachandran K.*, König M.*, Scharm M., Nguyen T.V.N, Hermjakob H., Waltemath D., Malik Sheriff R.S.</td>
      <td>Preprints 2022, 2022060137 (doi: 10.20944/preprints202206.0137.v1).</td>
      <td>2022-06-09 00:00:00</td>
      <td>Yes</td>
      <td>Matthias König</td>
      <td>P3 - König</td>
      <td>NaN</td>
      <td>10.20944/preprints202206.0137.v1</td>
    </tr>
    <tr>
      <td>Simvastatin therapy in different subtypes of hypercholesterolemia - a physiologically based modelling approach</td>
      <td>Bartsch F. , Grzegorzewski J., Pujol H., Tautenhahn HM., König M.</td>
      <td>medRxiv 2023.02.01.23285358; doi: https://doi.org/10.1101/2023.02.01.23285358</td>
      <td>2023-02-03 00:00:00</td>
      <td>Yes</td>
      <td>Jan Grzegorzewski, Hans-Michael Tautenhahn, Matthias König</td>
      <td>P3 - König, P9 - Tautenhahn</td>
      <td>NaN</td>
      <td>10.1101/2023.02.01.23285358</td>
    </tr>
    <tr>
      <td>standard-GEM: standardization of open-source genome-scale metabolic models</td>
      <td>Mihail Anton, Eivind Almaas, Rui Benfeitas, Sara Benito-Vaquerizo, Lars M. Blank, Andreas Dräger, John M. Hancock, Cheewin Kittikunapong, &lt;b&gt;Matthias König&lt;/b&gt;, Feiran Li, Ulf W. Liebal, Hongzhong Lu, Hongwu Ma, Radhakrishnan Mahadevan, Adil Mardinoglu, Jens Nielsen, Juan Nogales, Marco Pagni, Jason A. Papin, Kiran Raosaheb Patil, Nathan D. Price, Jonathan L. Robinson, Benjamín J. Sánchez, Maria Suarez-Diez, Snorre Sulheim, L. Thomas Svensson, Bas Teusink, Wanwipa Vongsangnak, Hao Wang, Ahmad A. Zeidan, Eduard J. Kerkhoven</td>
      <td>bioRxiv 2023.03.21.512712; doi: https://doi.org/10.1101/2023.03.21.512712</td>
      <td>2023-03-27 00:00:00</td>
      <td>No</td>
      <td>Matthias König</td>
      <td>P3 - König</td>
      <td>NaN</td>
      <td>10.1101/2023.03.21.512712</td>
    </tr>
    <tr>
      <td>A physiologically based pharmacokinetic model for CYP2E1 phenotyping via chlorzoxazone</td>
      <td>J. Küttner, J. Grzegorzewski, HM. Tautenhahn, M. König</td>
      <td>bioRxiv 2023.04.12.536571; doi: https://doi.org/10.1101/2023.04.12.536571</td>
      <td>4/14/2023</td>
      <td>Yes</td>
      <td>Jan Grzegorzewski, Hans-Michael Tautenhahn, Matthias König</td>
      <td>P3 - König, P9 - Tautenhahn</td>
      <td>NaN</td>
      <td>10.1101/2023.04.12.536571</td>
    </tr>
    <tr>
      <td>Eleven Strategies for Making Reproducible Research and Open Science Training the Norm at Research Institutions</td>
      <td>Friederike E. Kohrs, Susann Auer, Alexandra Bannach-Brown, Susann Fiedler, Tamarinde Haven, Verena Heise, Constance Holman, Flavio Azevedo, René Bernard, Arnim Bleier, Nicole Bössel, Brian Cahill, Leyla Jael Castro, Adrian Ehrenhofer, Kristina Eichel, Maximilian Frank, Claudia Frick, Malte Friese, Anne Gärtner, Kerstin Gierend, David Joachim Grüning, Lena Hahn, Maren Hülsemann, Malika Ihle, Sabrina Illius, Laura König, Matthias König, Louisa Kulke, Anton Kutlin, Fritjof Lammers, David M.A. Mehler, Christoph Miehl, Anett Müller-Alcazar, Claudia Neuendorf, Helen Niemeyer, Florian Pargent, Aaron Peikert, Christina U. Pfeuffer, Robert Reinecke, Jan Philipp Röer, Jessica L. Rohmann, Alfredo Sánchez-Tójar, Stefan Scherbaum, Elena Sixtus, Lisa Spitzer, Vera Maren Straßburger, Marcel Weber, Clarissa Whitmire, Josephine Zerna, Dilara Zorbek, Philipp Zumstein, Tracey L. Weissgerber</td>
      <td>osfpreprints 2023.04.12.536571 doi: 10.31219/osf.io/kcvra</td>
      <td>5/28/2023</td>
      <td>No</td>
      <td>Matthias König</td>
      <td>P3 - König</td>
      <td>NaN</td>
      <td>10.31219/osf.io/kcvra</td>
    </tr>
    <tr>
      <td>Quantifying Fat Zonation in Liver Lobules: An Integrated Multiscale In-silico Model Combining Disturbed Microperfusion and Fat Metabolism via a Continuum-Biomechanical Bi-scale, Tri-phasic Approach</td>
      <td>Lena Lambers, Navina Waschinsky, Jana Schleicher, Matthias König, Hans-Michael Tautenhahn, Mohamed Albadry, Uta Dahmen and Tim Ricken</td>
      <td>Research Square</td>
      <td>2023-12-09 00:00:00</td>
      <td>Yes</td>
      <td>Lena Lambers, Matthias König, Hans-Michael Tautenhahn, Mohamed Albadry, Uta Dahmen, Tim Ricken</td>
      <td>P2 - Dahmen, P3 - König, P7 - Ricken, P9 - Tautenhahn</td>
      <td>NaN</td>
      <td>10.21203/rs.3.rs-3348101/v1</td>
    </tr>
  </tbody>
</table>

<h2>Theses (n=5)</h2>
<table class="dataframe" id="theses">
  <thead>
    <tr style="text-align: right;">
      <th>Category</th>
      <th>Title</th>
      <th>Authors</th>
      <th>Date</th>
      <th>Qualiperf</th>
      <th>Authors Qualiperf</th>
      <th>Projects</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Bachelor Thesis</td>
      <td>A physiologically based model of pravastatin - The role of genotypes and hepatic or renal impairment on the pharmacokinetics of pravastatin</td>
      <td>Helen Leal Pujol, Matthias König</td>
      <td>2022-05-03 00:00:00</td>
      <td>No</td>
      <td>Matthias König</td>
      <td>P3 - König</td>
    </tr>
    <tr>
      <td>Master thesis</td>
      <td>Coupling of macro and micro scale in a continuum-biomechanical model of the human liver using preCICE</td>
      <td>Fritz Otlinghaus, Steffen Gerhäusser, Ishaan Desai, Tim Ricken, Benjamin Uekermann</td>
      <td>2022-08-01 00:00:00</td>
      <td>No</td>
      <td>Steffen Gerhäusser, Tim Ricken</td>
      <td>P7 - Ricken</td>
    </tr>
    <tr>
      <td>PhD thesis</td>
      <td>Multiscale and multiphase modeling and numerical simulation of function-perfusion processes in the liver</td>
      <td>Lambers, Lena</td>
      <td>3/21/2023</td>
      <td>No</td>
      <td>Lena Lambers</td>
      <td>P7 - Ricken</td>
    </tr>
    <tr>
      <td>Bachelor Thesis</td>
      <td>A physiologically based pharmacokinetic (PBPK) model of the probe drug talinolol for the characterization of intestinal P-glycoprotein</td>
      <td>Beatrice Stemmer Mallol, Matthias König (supervisor)</td>
      <td>7/17/2023</td>
      <td>No</td>
      <td>Matthias König</td>
      <td>P3 - König</td>
    </tr>
    <tr>
      <td>PhD thesis</td>
      <td>Physiologically based pharmacokinetic (PBPK) modeling for dynamical liver function tests and CYP phenotyping</td>
      <td>Jan Grzegorzewski, Matthias König</td>
      <td>2023-09-10 00:00:00</td>
      <td>No</td>
      <td>Jan Grzegorzewski, Matthias König</td>
      <td>P3 - König</td>
    </tr>
  </tbody>
</table>

<h2>Presentations (n=33)</h2>
<table class="dataframe" id="presentations">
  <thead>
    <tr style="text-align: right;">
      <th>Title</th>
      <th>Authors</th>
      <th>Conference</th>
      <th>Date</th>
      <th>Qualiperf</th>
      <th>Authors Qualiperf</th>
      <th>Projects</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Impact of impaired perfusion on drug metabolism in diﬀerent species</td>
      <td>M. Albadry , A. Lupp , O. Dirsch , H. - M. Tautenhahn , R. Klopﬂeisch , V. Liška, U. Dahmen</td>
      <td>24. Chirurgische Forschungstage, CFT2021, https://forschungstage2021.de/</td>
      <td>2021-09-02 00:00:00</td>
      <td>Yes</td>
      <td>Uta Dahmen</td>
      <td>P2 - Dahmen</td>
    </tr>
    <tr>
      <td>Bayesian statistics for uncertainty quantification of a physiologically based model of indocyanine green liver function tests</td>
      <td>Höpfl Sebastian, König Matthias , Radde Nicole</td>
      <td>24te Chirurgische Forschungstage, CFT2021, https://forschungstage2021.de/</td>
      <td>2021-09-03 00:00:00</td>
      <td>Yes</td>
      <td>Sebastian Höpfl, Matthias König, Nicole Erika Radde</td>
      <td>P5 - Radde</td>
    </tr>
    <tr>
      <td>Comparison of pCASL, DSC and FAIR for Cerebral Perfusion in Rats @9.4T</td>
      <td>Hung LY1, Herrmann KH, Renat Sibgatulin1, Hirschler L2,3, Warnking Jan3, Barbier EL3, Witte EL4, Holthoff K4, Reichenbach JR1, and Joerk A4.</td>
      <td>ISMRM</td>
      <td>2021-11-10 00:00:00</td>
      <td>Yes</td>
      <td>Hung LY1, Karl-Heinz Herrmann</td>
      <td>P6 - Reichenbach</td>
    </tr>
    <tr>
      <td>Fast 3D Isotropic High-Resolution MRI of Mouse Brain Using a Variable Flip Angle RARE Sequence With T2 Compensation @9.4T</td>
      <td>Herrman KH , Hung LY, and Reichenbach JR</td>
      <td>NaN</td>
      <td>2021-11-10 00:00:00</td>
      <td>Yes</td>
      <td>Karl-Heinz Herrman, Hung LY, Jürgen R Reichenbach</td>
      <td>P6 - Reichenbach</td>
    </tr>
    <tr>
      <td>A Multiscale and Multiphase, Data- and Knowledge-Driven Simulation of Function-Perfusion Processes in the Human Liver</td>
      <td>Lena Lambers, Steffen Gerhäusser, Luis Mandl, André Mielke, Tim Ricken</td>
      <td>The 8th European Congress on Computational Methods in Applied Sciences and Engineering ECCOMAS Congress 2022</td>
      <td>2022-06-05 00:00:00</td>
      <td>No</td>
      <td>Lena Lambers, Steffen Gerhäusser, Tim Ricken</td>
      <td>P7 - Ricken, Coordination</td>
    </tr>
    <tr>
      <td>Data- and Knowledge-Driven Multiphase and Multiscale Simulation of Growth and Remodeling Processes in the Human Liver</td>
      <td>Lena Lambers, Steffen Gerhäusser, Luis Mandl, André Mielke, Tim Ricken</td>
      <td>15th World Congress on Computational Mechanics (WCCM-XV)\n8th Asian Pacific Congress on Computational Mechanics (APCOM-VIII)</td>
      <td>2022-07-31 00:00:00</td>
      <td>No</td>
      <td>Lena Lambers, Steffen Gerhäusser, Tim Ricken</td>
      <td>P7 - Ricken, Coordination</td>
    </tr>
    <tr>
      <td>Prehabilitation in geriatric surgery</td>
      <td>Hans-Michael Tautenhahn</td>
      <td>Jahrestagung der mitteldeutschen Viszeralmedizin</td>
      <td>6/10/0022</td>
      <td>No</td>
      <td>Hans-Michael Tautenhahn</td>
      <td>P9 - Tautenhahn</td>
    </tr>
    <tr>
      <td>A Multiscale and Multiphase, Data- and Knowledge-Driven Simulation of Function-Perfusion Processes in the Human Liver</td>
      <td>Lena Lambers, Tim Ricken, Steffen Gerhäusser, Luis Mandl, André Mielke</td>
      <td>BioMechBW Symposium, 19.07.2022, Tübingen</td>
      <td>2022-07-19 00:00:00</td>
      <td>Yes</td>
      <td>Lena Lambers, Tim Ricken, Steffen Gerhäusser</td>
      <td>P7 - Ricken, Coordination</td>
    </tr>
    <tr>
      <td>Data- and Knowledge-Driven Multiphase and Multiscale Simulation of</td>
      <td>Lena Lambers, Steffen Gerhäusser, Luis Mandl, André Mielke\nand Tim Ricken</td>
      <td>92nd Annual Meeting of the International Association of Applied Mathematics and Mechanics (GAMM)</td>
      <td>2022-08-16 00:00:00</td>
      <td>Yes</td>
      <td>Lena Lambers, Steffen Gerhäusser, Tim Ricken</td>
      <td>P7 - Ricken</td>
    </tr>
    <tr>
      <td>Numerical Data- and Knowledge-Driven Simulation of Function-Perfusion Processes in Liver Cell, Tissue and Organ</td>
      <td>Lena Lambers, Steffen Gerhäusser, Luis Mandl, André Mielke, Tim Ricken</td>
      <td>GACM Congress, Essen</td>
      <td>2022-09-22 00:00:00</td>
      <td>Yes</td>
      <td>Lena Lambers, Steffen Gerhäusser, Tim Ricken</td>
      <td>P7 - Ricken</td>
    </tr>
    <tr>
      <td>Physiologically based pharmacokinetic (PBPK) models for metabolic phenotyping and evaluation of liver function</td>
      <td>König M, Grzegorzewski J</td>
      <td>ICSB2022, The 21st International Conference on Systems Biology</td>
      <td>2022-09-27 00:00:00</td>
      <td>Yes</td>
      <td>Matthias König, Jan Grzegorzewski</td>
      <td>P3 - König</td>
    </tr>
    <tr>
      <td>Modeling of function-perfusion interaction in the human liver based on a multiscale porous media model</td>
      <td>Steffen Gerhäusser, Tim Ricken, Lena Lambers, Luis Mandl, Matthias König</td>
      <td>GAMM Annual Meeting 2022 ,https://jahrestagung.gamm-ev.de/</td>
      <td>2022-08-18 00:00:00</td>
      <td>Yes</td>
      <td>Steffen Gerhäusser, Tim Ricken, Lena Lambers, Matthias König</td>
      <td>P7 - Ricken</td>
    </tr>
    <tr>
      <td>F.A.I.R. Bayesian workflows and F.A.I.R. model citation analysis</td>
      <td>Nicole Radde</td>
      <td>Combine Meeting, Satellite Event to the ICSB Conference, October 6-7, 2022, Berlin</td>
      <td>10/6/0022</td>
      <td>No</td>
      <td>Nicole Erika Radde</td>
      <td>P5 - Radde</td>
    </tr>
    <tr>
      <td>Bayesian modelling of enzymatic processes and model-based design of experiments</td>
      <td>Nicole Radde</td>
      <td>3rd EnzymeML Workshop, October 18-19, 2022, Rüdesheim</td>
      <td>10/18/0022</td>
      <td>No</td>
      <td>Nicole Erika Radde</td>
      <td>P5 - Radde</td>
    </tr>
    <tr>
      <td>Bayesian estimation shows the merits of reproducible and reusable modeling in systems biology</td>
      <td>Höpfl Sebastian, Radde Nicole</td>
      <td>BioSB 2023, https://www.aanmelder.nl/biosb2023, Egmond aan Zee</td>
      <td>2023-05-09 00:00:00</td>
      <td>Yes</td>
      <td>Sebastian Höpfl, Nicole Erika Radde</td>
      <td>P5 - Radde</td>
    </tr>
    <tr>
      <td>Journals in Systems Biology could improve their impact factor by enforcing reproducibility</td>
      <td>Höpfl Sebastian, Radde Nicole</td>
      <td>Workshop on Computational Models in Biology and Medicine 2023, http://www.biometrische-gesellschaft.de/arbeitsgruppen/statistische-methoden-i-d-bioinformatik/workshop2023.html</td>
      <td>2023-06-15 00:00:00</td>
      <td>Yes</td>
      <td>Sebastian Höpfl, Nicole Erika Radde</td>
      <td>P5 - Radde</td>
    </tr>
    <tr>
      <td>Outlier Detection in Steatosis Data using Bayesian Uncertainty Quantification via FAIR Modelling Tools</td>
      <td>Höpfl Sebastian, Albadry Mohamed, Dahmen Uta, Böttcher Michael and Radde Nicole</td>
      <td>International Conference on Systems Biology of Human Disease, https://www.sbhdberlin.org/index.html</td>
      <td>2023-06-19 00:00:00</td>
      <td>No</td>
      <td>Sebastian Höpfl, Mohamed Albadry, Uta Dahmen, Nicole Erika Radde</td>
      <td>P5 - Radde</td>
    </tr>
    <tr>
      <td>Reproducible models in Systems Biology are higher cited</td>
      <td>Höpfl Sebastian, Radde Nicole</td>
      <td>31st Conference on Intelligent Systems for Molecular Biology (ISMB), https://www.iscb.org/ismbeccb2023</td>
      <td>2023-07-23 00:00:00</td>
      <td>Yes</td>
      <td>Sebastian Höpfl, Nicole Erika Radde</td>
      <td>P5 - Radde</td>
    </tr>
    <tr>
      <td>FAIR Bayesian Optimal Experimental Design in Systems Biology</td>
      <td>Höpfl Sebastian, Pleiss Jürgen, Range Jan, Radde Nicole</td>
      <td>International Conference on Data-Integrated Simulation Science (SimTech2023), https://www.simtech2023.uni-stuttgart.de/</td>
      <td>2023-10-04 00:00:00</td>
      <td>Yes</td>
      <td>Sebastian Höpfl, Nicole Erika Radde</td>
      <td>P5 - Radde</td>
    </tr>
    <tr>
      <td>Hybrid Data- and Knowledge-Driven Multiscale and Multiphase Simulation of the Human Liver</td>
      <td>Lena Lambers, Tim Ricken, Steffen Gerhäusser, Luis Mandl, André Mielke</td>
      <td>SIAM CSE, Amsterdam, 26.02.2023 - 03.03.2023</td>
      <td>2023-03-03 00:00:00</td>
      <td>Yes</td>
      <td>Lena Lambers, Tim Ricken, Steffen Gerhäusser, Luis Mandl</td>
      <td>P7 - Ricken, Coordination</td>
    </tr>
    <tr>
      <td>Multiscale and Multiphase Modeling of Function-Perfusion Processes in the Liver on Organ, Lobule and Cell Scale</td>
      <td>Lena Lambers, Tim Ricken, Steffen Gerhäusser, Luis Mandl</td>
      <td>GAMM Fachausschuss Computational Biomechanics, 13.-14.02.2023 in Saarbrücken</td>
      <td>2023-02-13 00:00:00</td>
      <td>Yes</td>
      <td>Lena Lambers, Tim Ricken, Steffen Gerhäusser, Luis Mandl</td>
      <td>P7 - Ricken, Coordination</td>
    </tr>
    <tr>
      <td>Digital twins for liver function and metabolic phenotyping</td>
      <td>Matthias König</td>
      <td>MBM2023 - Workshop on Modelling in Biology and Medicine - https://mbm.systemsbiology.se</td>
      <td>5/17/2023</td>
      <td>Yes</td>
      <td>Matthias König</td>
      <td>P3 - König</td>
    </tr>
    <tr>
      <td>Reproducible models in Systems Biology are higher cited</td>
      <td>Höpfl Sebastian, Radde Nicole</td>
      <td>ISMB/ECCB 2023 Conference on Intelligent Systems for Molecular Biology (ISMB) and European Conference on Computational Biology (ECCB)</td>
      <td>7/24/2023</td>
      <td>Yes</td>
      <td>Sebastian Höpfl, Nicole Erika Radde</td>
      <td>P5 - Radde</td>
    </tr>
    <tr>
      <td>Simulation Supported Liver Assessment for Donor Organs (SimLivA) - Continuum-Biomechanical Modeling for Staging of Ischemia Reperfusion Injury During Liver Transplantation</td>
      <td>Mandl L., Gerhäusser S., Lambers L., König M., Tautenhahn H.-M., Dahmen U., Ricken, T.</td>
      <td>93. Jahrestagung der Gesellschaft für angewandte Mathematik und Mechanik; https://jahrestagung.gamm-ev.de/</td>
      <td>2023-02-06 00:00:00</td>
      <td>Yes</td>
      <td>Luis Mandl, Steffen Gerhäusser, Lena Lambers, Matthias König, Hans-Michael Tautenhahn, Uta Dahmen, Tim Ricken</td>
      <td>P2 - Dahmen, P3 - König, P7 - Ricken, P9 - Tautenhahn</td>
    </tr>
    <tr>
      <td>Multiscale and Multiphase Simulation of Function- Perfusion Processes in the Liver on Whole Body, Organ, Lobule, and Cell Scale</td>
      <td>L. Lambers, S. Gerhäusser, L. Mandl, T. Ricken</td>
      <td>93. Jahrestagung der Gesellschaft für angewandte Mathematik und Mechanik (GAMM), Dresden</td>
      <td>5/30/2023</td>
      <td>Yes</td>
      <td>Lena Lambers, Steffen Gerhäusser, Luis Mandl, Tim Ricken</td>
      <td>P7 - Ricken, SimLivA, ATLAS</td>
    </tr>
    <tr>
      <td>Advancing Liver Function Assessment: Personalized and Stratified Approaches with Standardized Computational Models and Data</td>
      <td>Matthias König</td>
      <td>Workshop on Computational Models in Biology and Medicine 2023</td>
      <td>6/16/2023</td>
      <td>Yes</td>
      <td>Matthias König</td>
      <td>P3 - König, SimLivA, ATLAS</td>
    </tr>
    <tr>
      <td>Simulation Supported Liver Assessment for Donor Organs (SimLivA) - Con- tinuum-Biomechanical Modeling for Staging of Ischemia Reperfusion Injury During Liver Transplantation</td>
      <td>Mandl L., Gerhäusser S., Lambers L., König M., Tautenhahn H.-M., Dahmen U., Ricken, T.</td>
      <td>93rd Annual Meeting of the International Association of Applied Mathematics and Mechanics, https://jahrestagung.gamm.org/annual-meeting-2023</td>
      <td>2023-02-06 00:00:00</td>
      <td>Yes</td>
      <td>Luis Mandl, Steffen Gerhäusser, Lena Lambers, Matthias König, Hans-Michael Tautenhahn, Uta Dahmen, Tim Ricken</td>
      <td>P2 - Dahmen, P3 - König, P7 - Ricken, P9 - Tautenhahn, SimLivA</td>
    </tr>
    <tr>
      <td>SIMulation Supported LIVer Assessment for Donor Organs (SimLivA) - Continuum-Biomechanical Modeling for Staging of Ischemia Reperfusion Injury during Liver Transplantation</td>
      <td>Mandl L., Bütow L., König M., Dahmen U., Chen X., Tautenhahn H.-M., Ricken T.</td>
      <td>SPP2311 - Annual Workshop Magdeburg</td>
      <td>2023-12-09 00:00:00</td>
      <td>No</td>
      <td>Luis Mandl, Laura Bütow, Matthias König, Uta Dahmen, Hans-Michael Tautenhahn, Tim Ricken</td>
      <td>P2 - Dahmen, P3 - König, P7 - Ricken, P9 - Tautenhahn, SimLivA</td>
    </tr>
    <tr>
      <td>Staging of ischemia reperfusion injury during liver transplantation using continuum-biomechanical modeling</td>
      <td>Mandl L., Gerhäusser S., Lambers L., König M., Tautenhahn H.-M., Dahmen U., Ricken T.</td>
      <td>X International Conference on Computational Bioengineering, Wien, 20.-22.09.2023, https://iccb2023.conf.tuwien.ac.at/</td>
      <td>9/22/2023</td>
      <td>Yes</td>
      <td>Luis Mandl, Steffen Gerhäusser, Lena Lambers, Matthias König, Hans-Michael Tautenhahn, Uta Dahmen, Tim Ricken</td>
      <td>P2 - Dahmen, P3 - König, P7 - Ricken, P9 - Tautenhahn, SimLivA</td>
    </tr>
    <tr>
      <td>Digital twins for liver function &amp; metabolic phenotyping</td>
      <td>Matthias König</td>
      <td>AstraZeneca Seminar</td>
      <td>2023-01-09 00:00:00</td>
      <td>Yes</td>
      <td>Matthias König</td>
      <td>P3 - König, SimLivA, ATLAS</td>
    </tr>
    <tr>
      <td>Studying the impact of drug metabolism on the liver tissue: An integrated biomechanical modeling approach</td>
      <td>Matthias König, Steffen Gerhäusser, Luis Mandl, Lena Lambers, Uta Dahmen, Hans-Michael Tautenhahn, Tim Ricken</td>
      <td>ICCB2023 - X International Conference on Computational Bioengineering</td>
      <td>9/22/2023</td>
      <td>Yes</td>
      <td>Matthias König, Steffen Gerhäusser, Luis Mandl, Lena Lambers, Uta Dahmen, Hans-Michael Tautenhahn, Tim Ricken</td>
      <td>P2 - Dahmen, P3 - König, P7 - Ricken, P9 - Tautenhahn, SimLivA, ATLAS</td>
    </tr>
    <tr>
      <td>Reproducible digital twins for personalized liver function assessment</td>
      <td>Matthias König</td>
      <td>ITB - Institute for Theoretical Biology, Humboldt-University Berlin</td>
      <td>2023-10-10 00:00:00</td>
      <td>Yes</td>
      <td>Matthias König</td>
      <td>P3 - König, SimLivA, ATLAS</td>
    </tr>
    <tr>
      <td>Reproducible digital twins for personalized liver function assessment</td>
      <td>Matthias König</td>
      <td>e:Med Meeting 2023 on Systems Medicine</td>
      <td>2023-10-10 00:00:00</td>
      <td>Yes</td>
      <td>Matthias König</td>
      <td>P3 - König, SimLivA, ATLAS</td>
    </tr>
  </tbody>
</table>

<h2>Posters (n=21)</h2>
<table class="dataframe" id="posters">
  <thead>
    <tr style="text-align: right;">
      <th>Title</th>
      <th>Authors</th>
      <th>Conference</th>
      <th>Date</th>
      <th>Qualiperf</th>
      <th>Authors Qualiperf</th>
      <th>Projects</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Differential gene expression analysis to determine liver perfusion and liver function</td>
      <td>Daria Meyer, Manja Marz</td>
      <td>24. Chirurgische Forschungstage, CFT2021, https://forschungstage2021.de/</td>
      <td>2022-09-02 00:00:00</td>
      <td>Yes</td>
      <td>Daria Meyer, Manja Marz</td>
      <td>P4 - Marz</td>
    </tr>
    <tr>
      <td>An In-silico Model of Coupled Function-Perfusion Processes in the Human Liver with Application to Steatosis and Tumor Growth</td>
      <td>Lambers L, Ricken T</td>
      <td>24te Chirurgische Forschungstage, CFT2021, https://forschungstage2021.de/</td>
      <td>2021-09-02 00:00:00</td>
      <td>Yes</td>
      <td>Tim Ricken</td>
      <td>P7 - Ricken</td>
    </tr>
    <tr>
      <td>F.A.I.R. Workflows for Bayesian Inference in Systems Medicine</td>
      <td>Höpfl Sebastian, Radde Nicole E.</td>
      <td>8th international conference on Systems Biology of Mammalian Cells SBMC 2022, https://sbmc2022.de/</td>
      <td>2022-05-17 00:00:00</td>
      <td>Yes</td>
      <td>Sebastian Höpfl, Nicole Erika Radde</td>
      <td>P5 - Radde</td>
    </tr>
    <tr>
      <td>Data and Model Driven Multiscale Simulation of Tumor Growth in the Human Liver</td>
      <td>Lena Lambers, Tim Ricken, Arndt Wagner, André Mielke, Marlon Suditsch</td>
      <td>SimTech Status Seminar</td>
      <td>2021-10-12 00:00:00</td>
      <td>Yes</td>
      <td>Lena Lambers, Tim Ricken</td>
      <td>P7 - Ricken, Coordination</td>
    </tr>
    <tr>
      <td>Data- and Model-Driven Multiscale Simulation of Tumor Growth in the Human Liver</td>
      <td>Lena Lambers, Tim Ricken, Steffen Gerhäusser, Luis Mandl</td>
      <td>SimTech Status Seminar 2022, Bad Boll</td>
      <td>7/5/0022</td>
      <td>Yes</td>
      <td>Lena Lambers, Tim Ricken, Steffen Gerhäusser</td>
      <td>P7 - Ricken, Coordination</td>
    </tr>
    <tr>
      <td>Data- and model-driven multiscale liver modelling function-perfusion interqaction during liver resction</td>
      <td>Gerhäusser S., Ricken T., Lambers L., Mandl L.</td>
      <td>Simtech Status Seminar 2022</td>
      <td>2022-07-04 00:00:00</td>
      <td>Yes</td>
      <td>Steffen Gerhäusser, Tim Ricken, Lena Lambers</td>
      <td>P7 - Ricken, Coordination</td>
    </tr>
    <tr>
      <td>Data- and Model-Driven Multiscale Liver Modelling - Staging Ischemia Reperfusion Injury During Liver Transplantation</td>
      <td>Luis Mandl, Tim Ricken, Lena Lambers, Steffen Gerhäusser</td>
      <td>SimTech Status-Seminar Bad Boll 2022</td>
      <td>2022-07-04 00:00:00</td>
      <td>Yes</td>
      <td>Tim Ricken, Lena Lambers, Steffen Gerhäusser</td>
      <td>P7 - Ricken, Coordination</td>
    </tr>
    <tr>
      <td>Data- and Model-Driven Multiscale Liver Modelling</td>
      <td>Lena Lambers, Tim Ricken</td>
      <td>Pints&amp;Posters, 15.07.2022, University of Stuttgart</td>
      <td>2022-07-15 00:00:00</td>
      <td>Yes</td>
      <td>Lena Lambers, Tim Ricken</td>
      <td>P7 - Ricken, Coordination</td>
    </tr>
    <tr>
      <td>F.A.I.R. Workflows for Bayesian Inference in Systems Medicine</td>
      <td>Höpfl Sebastian, Radde Nicole</td>
      <td>SimTech Status Seminar 2022, Bad Boll</td>
      <td>2022-07-04 00:00:00</td>
      <td>Yes</td>
      <td>Sebastian Höpfl, Nicole Erika Radde</td>
      <td>P5 - Radde</td>
    </tr>
    <tr>
      <td>F.A.I.R. Workflows for Bayesian Inference in Systems Medicine</td>
      <td>Sebastian Höpfl, Nicole E. Radde</td>
      <td>Current Trends in Systems and Control Theory, June 7 2022 Stuttgart</td>
      <td>2022-06-07 00:00:00</td>
      <td>Yes</td>
      <td>Sebastian Höpfl, Nicole Erika Radde</td>
      <td>P5 - Radde</td>
    </tr>
    <tr>
      <td>Differences in gene expression are stronger influenced by patient's sex than by biological conditions like fat content or disease</td>
      <td>Meyer D, Barth E, Marz M</td>
      <td>German Conference on Bioinformatics 2022, https://gcb2022.de/</td>
      <td>2022-09-06 00:00:00</td>
      <td>Yes</td>
      <td>Daria Meyer, Manja Marz</td>
      <td>P4 - Marz</td>
    </tr>
    <tr>
      <td>Digital Liver Twin</td>
      <td>Ricken Tim, Gerhäusser Steffen, Lambers Lena, Mandl Luis</td>
      <td>Inventing the Future - University of Stuttgart meets Bosch</td>
      <td>2022-10-24 00:00:00</td>
      <td>Yes</td>
      <td>Ricken Tim, Gerhäusser Steffen, Lambers Lena</td>
      <td>P7 - Ricken</td>
    </tr>
    <tr>
      <td>Asymmetric echo variable flip angle RARE for n-echo Dixon based PDFF @ 9.4T</td>
      <td>Zhao WT, Herrmann KH, Sibgatulin R, Arlt J, Wei WW, Dahmen U, and Reichenbach JR.</td>
      <td>International Society for Magnetic Resonance in Medicine 2023, https://www.ismrm.org/23m/</td>
      <td>2023-02-03 00:00:00</td>
      <td>Yes</td>
      <td>Wan-Ting Zhao, Karl-Heinz Herrmann, Sibgatulin R, Weiwei Wei, Uta Dahmen, Jürgen R. and Reichenbach</td>
      <td>P2 - Dahmen, P6 - Reichenbach</td>
    </tr>
    <tr>
      <td>Digital Liver Twin</td>
      <td>Lena Lambers, Tim Ricken, Steffen Gerhäusser, Luis Mandl</td>
      <td>Treffen der Joachim-Herz-Stiftung, 10.-11.20.2023 in Hamburg</td>
      <td>2023-02-11 00:00:00</td>
      <td>Yes</td>
      <td>Lena Lambers, Tim Ricken, Steffen Gerhäusser, Luis Mandl</td>
      <td>P7 - Ricken, Coordination</td>
    </tr>
    <tr>
      <td>The influence of marginal organs on the drug metabolism in liver transplantation</td>
      <td>Bütow L.B., Tautenhahn H.-M. T., Schröter D.S., Schlattmann P.S., Settmacher U.S., Dahmen U.D.,</td>
      <td>EASL European Association for the Study of the Liver, https://www.easlcongress.eu,</td>
      <td>6/24/2023</td>
      <td>Yes</td>
      <td>Laura Bütow, Hans-Michael Tautenhahn. T., Schröter D.S., Uta Dahmen</td>
      <td>P2 - Dahmen, P9 - Tautenhahn, SimLivA</td>
    </tr>
    <tr>
      <td>Influence of marginal organs on the drug metabolism in liver transplantation. Outline for a prospective clinical study</td>
      <td>Bütow L.B., Schröter D.S., Tautenhahn H.-M. T., Dahmen U.D., Nickel S. N.,</td>
      <td>33., Wilseder Workshop von Biotest- Individulaisierte Therapie von Lebererkrnakungen, https://www.biotest-wilsede.de,</td>
      <td>6/28/2023</td>
      <td>Yes</td>
      <td>Laura Bütow, Schröter D.S., Hans-Michael Tautenhahn, Uta Dahmen</td>
      <td>P2 - Dahmen, P9 - Tautenhahn, SimLivA</td>
    </tr>
    <tr>
      <td>Time-series of transcriptome data reveal non-coding RNAs can be incorporated into modelling approaches</td>
      <td>Meyer D., Barth E., Christ B., König M., Marz M.</td>
      <td>Workshop on Computational Models in Biology and Medicine 2023</td>
      <td>6/15/2023</td>
      <td>Yes</td>
      <td>Daria Meyer, Bruno Christ, Matthias König, Manja Marz</td>
      <td>P1 - Christ, P3 - König, P4 - Marz</td>
    </tr>
    <tr>
      <td>Digital Liver Twin - Data- and Model-Driven Multiscale Liver Modeling</td>
      <td>Tim Ricken, Steffen Gerhäusser, Lena Lambers, Luis Mandl</td>
      <td>International Conference on Data-Integrated Simulation Science (SimTech2023), https://www.simtech2023.uni-stuttgart.de/, 04. - 06. Oktober 2023, Sutttgart</td>
      <td>2023-05-10 00:00:00</td>
      <td>Yes</td>
      <td>Tim Ricken, Steffen Gerhäusser, Lena Lambers, Luis Mandl</td>
      <td>P7 - Ricken</td>
    </tr>
    <tr>
      <td>GlucoCEST MRI to Assess Alterations in the Corticomedullary Glucose Gradient in a Rat Model of S-AKI</td>
      <td>Wan-Ting Zhao, Karl-Heinz Herrmann, Jürgen R. Reichenbach and Verena Hoerr\n</td>
      <td>25th DS-ISMRM (Germany chapter International Society for Magnetic Resonance in Medicine), Berlin, 6-7 September, 2023, Poster</td>
      <td>None</td>
      <td>None</td>
      <td>None</td>
      <td>None</td>
    </tr>
    <tr>
      <td>Adaptive Response of Liver Perfusion in Preclinical Portal Vein Ligation</td>
      <td>Wan-Ting Zhao, Weiwei Wei, Karl-Heinz Herrmann, Martin Krämer, Uta Dahmen, Jürgen R. Reichenbach</td>
      <td>39th European Society for Magnetic Resonance in Medicine and Biology, Basel, Switzerland. Poster</td>
      <td>None</td>
      <td>None</td>
      <td>None</td>
      <td>None</td>
    </tr>
    <tr>
      <td>FAIR Bayesian Optimal Experimental Design in Systems Biology</td>
      <td>Höpfl Sebastian, Wagner Vincent, Guizeni Eya, Pleiss Jürgen, Range Jan, Radde Nicole</td>
      <td>International Conference on Data-Integrated Simulation Science (SimTech2023), https://www.simtech2023.uni-stuttgart.de/</td>
      <td>2023-06-10 00:00:00</td>
      <td>Yes</td>
      <td>Sebastian Höpfl, Nicole Erika Radde</td>
      <td>P5 - Radde</td>
    </tr>
  </tbody>
</table>

<h2>Others (n=26)</h2>
<table class="dataframe" id="others">
  <thead>
    <tr style="text-align: right;">
      <th>Category</th>
      <th>Title</th>
      <th>Authors</th>
      <th>Date</th>
      <th>Qualiperf</th>
      <th>Projects</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Editoring</td>
      <td>Computational Modeling for Liver Surgery and Interventions</td>
      <td>Bruno Christ, Uta Dahmen, Nicole Radde, Tim Ricken</td>
      <td>2021-07-31 00:00:00</td>
      <td>No</td>
      <td>P1 - Christ, P2 - Dahmen, P5 - Radde, P7 - Ricken</td>
    </tr>
    <tr>
      <td>Organization Minisymposium</td>
      <td>Computational Continuum Biomechanics</td>
      <td>Tim Ricken, Oliver Röhrle, Silvia Budday</td>
      <td>2022-07-31 00:00:00</td>
      <td>No</td>
      <td>P7 - Ricken</td>
    </tr>
    <tr>
      <td>Organization Minisymposium</td>
      <td>CONTINUUM BIOMECHANICS OF ACTIVE SYSTEMS</td>
      <td>Tim Ricken, Oliver Röhrle, Silvia Budday</td>
      <td>2022-06-05 00:00:00</td>
      <td>No</td>
      <td>P7 - Ricken</td>
    </tr>
    <tr>
      <td>Editoring</td>
      <td>Frontiers Research Topic “Multiscale Modeling for the Liver”</td>
      <td>Ho H, Rezaina V, Schwen LO</td>
      <td>2021-11-16 00:00:00</td>
      <td>No</td>
      <td>P8 - Schwen</td>
    </tr>
    <tr>
      <td>Organization Minisymposium</td>
      <td>Computational biomechanics and biomedical engineering of active biological systems – from methods to clinical application</td>
      <td>Christian Bleiler, Lena Lambers, Renate Sachse</td>
      <td>2022-09-21 00:00:00</td>
      <td>No</td>
      <td>P7 - Ricken, Coordination</td>
    </tr>
    <tr>
      <td>Editoring</td>
      <td>Progress in Liver Stem Cell Therapy</td>
      <td>Christ B, Oertel M</td>
      <td>2022-05-31 00:00:00</td>
      <td>No</td>
      <td>P1 - Christ</td>
    </tr>
    <tr>
      <td>Accepted project proposal</td>
      <td>Michael Stifel grant</td>
      <td>Lena Lambers, Uta Dahmen</td>
      <td>2022-05-31 00:00:00</td>
      <td>Yes</td>
      <td>P2 - Dahmen, Coordination</td>
    </tr>
    <tr>
      <td>Workshop</td>
      <td>Workshop on Computational Models in Biology and Medicine</td>
      <td>Nicole Radde, Sebastian Höpfl</td>
      <td>2022-06-24 00:00:00</td>
      <td>No</td>
      <td>P5 - Radde</td>
    </tr>
    <tr>
      <td>Miscellaneous</td>
      <td>Kongresssekretär der Jahrestagung der mittelseutschen Viszeralmedizin</td>
      <td>Hans-Michael Tautenhahn</td>
      <td>6/10/0022</td>
      <td>No</td>
      <td>P9 - Tautenhahn</td>
    </tr>
    <tr>
      <td>Miscellaneous</td>
      <td>Vorstandsmitglied der Thüringischen Gesellschaft für Chirurgie</td>
      <td>Hans-Michael Tautenhahn</td>
      <td>6/1/0022</td>
      <td>No</td>
      <td>P9 - Tautenhahn</td>
    </tr>
    <tr>
      <td>Miscellaneous</td>
      <td>New Project Assistant</td>
      <td>Hans-Michael Tautenhahn</td>
      <td>8/1/0022</td>
      <td>No</td>
      <td>P9 - Tautenhahn</td>
    </tr>
    <tr>
      <td>Workshop Leader</td>
      <td>GmdS/IBS Arbeitsgruppe Mathematical Models in Medicine and Biology</td>
      <td>Nicole Radde, Ingmar Glauche</td>
      <td>2022-07-18 00:00:00</td>
      <td>No</td>
      <td>P5 - Radde</td>
    </tr>
    <tr>
      <td>Accepted project proposal</td>
      <td>Methodology for the calibration and analysis of stochastic models for heterogeneous intracellular processes with applications in cancer development</td>
      <td>Nicole Radde</td>
      <td>2022-08-01 00:00:00</td>
      <td>No</td>
      <td>P5 - Radde</td>
    </tr>
    <tr>
      <td>Accepted project proposal</td>
      <td>Data-enhanced prediction of organ-specific tumor growth in the liver - a hybrid knowledge and data-driven approach</td>
      <td>Tim Ricken</td>
      <td>2022-08-01 00:00:00</td>
      <td>No</td>
      <td>P7 - Ricken</td>
    </tr>
    <tr>
      <td>Editoring</td>
      <td>Editor-in-Chief der Zeitschrift "Zeitschrift für Medizinische Physik" (Journal of Medical Physics) (ISSN 0939-3889)</td>
      <td>Jürgen Reichenbach</td>
      <td>2021-11-01 00:00:00</td>
      <td>No</td>
      <td>P6 - Reichenbach</td>
    </tr>
    <tr>
      <td>Internship</td>
      <td>Computational Modeling of Drug Detoxification – A Systems Medicine Approach</td>
      <td>Matthias König</td>
      <td>2022-10-24 00:00:00</td>
      <td>No</td>
      <td>P3 - König</td>
    </tr>
    <tr>
      <td>Accepted project proposal</td>
      <td>SIMulation supported LIVer Assessment for donor organs (SimLivA) - Continuum-biomechanical modeling for staging of ischemia reperfusion injury during liver transplantation</td>
      <td>Professorin Dr. Uta Dahmen, Professor Dr.-Ing. Tim Ricken, Privatdozent Dr. Hans-Michael Tautenhahn, Dr. Matthias König</td>
      <td>2020-12-16 00:00:00</td>
      <td>No</td>
      <td>P2 - Dahmen, P3 - König, P7 - Ricken, P9 - Tautenhahn</td>
    </tr>
    <tr>
      <td>Accepted project proposal</td>
      <td>031L0304X - CompLS - Runde 5 - Verbundprojekt: ATLAS - Al and Simulation for Tumor Liver Assessment - Entwicklung eines Systems zur klinischen Entscheidungsunterstützung in der Diagnose und Behandlung von Lebertumoren auf Basis von künstlicher Intelligenz und Simulationen</td>
      <td>Professor Dr.-Ing. Tim Ricken, Professor Dr. Steffen Staab, Privatdozent Dr. Hans-Michael Tautenhahn, Dr. Matthias König</td>
      <td>2022-09-15 00:00:00</td>
      <td>No</td>
      <td>P3 - König, P7 - Ricken, P9 - Tautenhahn</td>
    </tr>
    <tr>
      <td>Editoring</td>
      <td>Elected PEtab editor from March 2023 until March 2026</td>
      <td>König Matthias</td>
      <td>2023-02-07 00:00:00</td>
      <td>No</td>
      <td>P3 - König, P5 - Radde</td>
    </tr>
    <tr>
      <td>Accepted project proposal</td>
      <td>X-Research Group: Physiologically based digital twins for the treatment of hypertension with ACE inhibitors and diuretics</td>
      <td>König Matthias</td>
      <td>2023-02-12 00:00:00</td>
      <td>No</td>
      <td>P3 - König</td>
    </tr>
    <tr>
      <td>Editoring</td>
      <td>M. König has been elected as PEtab editor (2023-2026)</td>
      <td>König Matthias</td>
      <td>2023-02-07 00:00:00</td>
      <td>No</td>
      <td>P3 - König</td>
    </tr>
    <tr>
      <td>Dataset</td>
      <td>Datasets for "Automated Detection of Portal Fields and Central Veins in Whole-Slide Images of Liver Tissue"</td>
      <td>Budelmann D, Laue H, Weis N, Dahmen U, D’Alessandro LA, Biermayer I, Klingmüller U, Ghallab A, Hassan R, Begher-Tibbe B, Hengstler JG, Schwen LO</td>
      <td>2021-08-12 00:00:00</td>
      <td>No</td>
      <td>Stea-PK Mod</td>
    </tr>
    <tr>
      <td>Dataset</td>
      <td>Dataset for "Segmentation of Lipid Droplets in Histological Images"</td>
      <td>Budelmann D, Cao Q, Laue H, Albadry M, Dahmen U, Schwen LO</td>
      <td>2023-05-04 00:00:00</td>
      <td>No</td>
      <td>Stea-PK Mod</td>
    </tr>
    <tr>
      <td>Hosting and organization of COMBINE 2024</td>
      <td>Hosting and organiztion of the Computational Modeling in Biology” Network (COMBINE) meeting 2024 in Stuttgart</td>
      <td>Radde Nicole, Waltemath Dagmar, Höpfl Sebastian</td>
      <td>2024-01-09 00:00:00</td>
      <td>Yes</td>
      <td>P5 - Radde</td>
    </tr>
    <tr>
      <td>Organization Minisymposium</td>
      <td>In-silico Models of Coupled Biological Systems</td>
      <td>Radde Nicole</td>
      <td>2023-05-10 00:00:00</td>
      <td>Yes</td>
      <td>P5 - Radde</td>
    </tr>
    <tr>
      <td>Invited Speaker</td>
      <td>Mathematische Modellbildung des Ischämie-Reperfusionsschadens zur Entscheidungsunterstützung bei Lebertransplantationen</td>
      <td>Mandl L., Gerhäusser S., Lambers L., König M., Tautenhahn H.-M., Dahmen U., Ricken T.</td>
      <td>10/28/2023</td>
      <td>Yes</td>
      <td>P2 - Dahmen, P3 - König, P7 - Ricken, P9 - Tautenhahn, SimLivA</td>
    </tr>
  </tbody>
</table>


<script>
    new DataTable('#statistics');
    new DataTable('#publications');
    new DataTable('#submissions');
    new DataTable('#preprints');
    new DataTable('#theses');
    new DataTable('#presentations');
    new DataTable('#posters');
    new DataTable('#others');

</script>

</body>

</html>
